### DISCUSSION TITLE
AFMF relies on DVR64

### DISCUSSION THREAD

**Discussion by TheJman494** [GSDragoon/RadeonSoftwareSlimmer#88] [General]
AMD Fluid Motion Frames officially released in v24.1.1 relies on the DVR64 component, might want to add a label for those who want to use it.

**Comment by usama7628674** [comment]
I install driver without DVR yet I still see AFMF option in adrenalin edition software. I haven't tested it yet, but I don't think DVR is required for AFMF to work.
